pkg/scheduler/objects/init.go

```diff
@@ -0,0 +1,37 @@
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package objects
+
+import (
+	"time"
+
+	"github.com/oklog/ulid/v2"
+	"golang.org/x/exp/rand"
+)
+
+var (
+	entropy *rand.Rand
+	ms      uint64
+)
+
+func init() {
+	entropy = rand.New(new(rand.LockedSource))
+	entropy.Seed(uint64(time.Now().UnixNano()))
+	ms = ulid.Timestamp(time.Now())
```

Comment:
Why not 😃
```suggestion
	ms = ulid.Timestamp(time.Now().UnixNano())
```

Reply:
The timestamp is not usually sufficient to be an entropy since it only has few bytes of entropy. However, we should just proceed with this change. The only thing important to us is that it has a locked source.

Reply:
Yeah, `UnixNano()` doesn’t really add any value here—it’s more for looks as the prev line has nano.